easybuild/easyconfigs/p/pyplusplus/pyplusplus-2c6c05b7a7d2-foss-2016a-Python-3.5.1.eb

```diff
@@ -0,0 +1,32 @@
+easyblock = 'PythonPackage'
+
+name = 'pyplusplus'
+version = '2c6c05b7a7d2'
```

Comment:
@hajgato wouldn't it be better to use a datestamp as version here (which sorts properly) rather than a commit ID?